src/visu_config.xsd

```diff
@@ -598,12 +598,42 @@
             <xsd:appinfo>level:expert</xsd:appinfo>
         </xsd:annotation>
     </xsd:attribute>
-    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="showfooter" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-top" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-bottom" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-left" type="xsd:boolean" use="optional" />
-    <xsd:attribute name="shownavbar-right" type="xsd:boolean" use="optional" />
+    <xsd:attribute name="showtopnavigation" type="xsd:boolean" use="optional">
+    	<xsd:annotation>
+            <xsd:documentation xml:lang="en">the navigation path shows on which subpage of the visu one currently is</xsd:documentation>
```

Comment:
The sentences are usually starting with a capital letter. And this is quite informal wording.

I'd suggest something like: Select the display of the top navigation that shows the currently displayed page with all parent pages.